var/spack/repos/builtin/packages/hsa-rocr-dev/package.py

```diff
@@ -61,7 +63,9 @@ class HsaRocrDev(CMakePackage):
     def cmake_args(self):
         spec = self.spec
 
-        libelf_include = spec['libelf'].prefix.include.libelf
+        libelf_include = os.path.dirname(
+            find_headers('libelf', spec['elf'].prefix.include, recursive=True)[0])
```

Comment:
```suggestion
        libelf_include = spec['elf'].headers.directories[0]
```

Can you verify this works for the other elf providers? This should be the way to go.